### ISSUE TITLE
[TB] Settings Slider - WCAG 4.1.2: Ensures every form element has a label (input[aria-valuemax="256"])

### ISSUE THREAD

**Issue by kburk1997** [finos/a11y-theme-builder#923]
## Problem/Concern

When I edit small text styles and run Accessibility Insights for Web, I get the following issue:
### Issue
`Ensures every form element has a label` ([label](https://accessibilityinsights.io/info-examples/web/label))
### Target application
[A11y ThemeBuilder](http://localhost:8080/designSystem/Sample)
### Element path
input[aria-valuemax="256"]
### Snippet
`<input data-index="0" aria-valuenow="16" aria-orientation="horizontal" aria-valuemax="256" aria-valuemin="0" type="range" min="0" max="256" step="1" value="16" style="border: 0px; clip: rect(0px, 0px, 0px, 0px); height: 100%; margin: -1px; overflow: hid...`
### How to fix
Fix any of the following:
  Form element does not have an implicit (wrapped) <label>
  Form element does not have an explicit <label>
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element has no placeholder attribute
  Element's default semantics were not overridden with role="none" or role="presentation"
### Environment
Microsoft Edge version 125.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.42.0 (axe-core 4.8.4), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
### Screenshot
<img width="703" alt="Screenshot of font size slider element where issue occurs" src="https://github.com/finos/a11y-theme-builder/assets/11298269/da6383c3-88f1-4d05-95a1-69214e581643">

## Proposed Solution
Ensure all sliders are properly labeled.

**Comment by kburk1997** [comment]
I am now covered under my company's CCLA, so I can take this on if needed - this seems like a very small fix.

**Comment by aaronreed708** [comment]
> I am now covered under my company's CCLA, so I can take this on if needed - this seems like a very small fix.

Sure, if you have the time to fix it, that would be great!  Then you can get a contributors badge! :-)

**Comment by kburk1997** [comment]
PR merged - closing
